### PR TITLE
Move recaptcha config to contact fragment frontmatter

### LINKED ISSUE #88: Set recaptcha sitekey via fragment front-matter
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
Currently we set recaptcha sitekey globally.

**What you expected to happen**:
Set sitekey per contact fragment.

**Anything else we need to know?**:
Still needs a discussion. Globally simplifies recaptchas, but it bundles site config with a fragment config. We decided to add scripts from fragments within each fragment. We should probably be consistent and also set fragment specific config to their respective front-matter. Aka sitekey.

Most users with one contact form won't notice. For bigger users this duplicates the config in various front-matters, but also enables various recaptcha keys.
Thoughts?

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
Move recaptcha config from main config file to fragment config

Also made recaptcha optional with one set-back; Submit button is always enabled.

**Which issue this PR fixes** 
fixes #88

### CODE REVIEW

File: exampleSite/content/index/contact.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ email = "mail@example.com"
 button = "Send Button" # defaults to theme default
 #netlify = false
 
+# Optional google captcha
+#[params.contact.captcha]
```
Comment (stp-ip) [inline exampleSite/content/index/contact.md]: Could probably be simplified to just:
```
#[captcha]
#  sitekey = ""
```
No need to have the more complex format.

Comment (stp-ip) [reply exampleSite/content/index/contact.md]: Or [recaptcha] to make it clear, that it's a setting for google's recaptcha.

File: static/main.css [hunk 2]
```diff
@@ -9691,9 +9691,11 @@ div.form-error ul li {
 
 #particles-js {
```
Comment (stp-ip) [inline static/main.css]: Separate change so might usually be better as a separate PR.

**Comment by mpourismaiel** [comment]
@stp-ip @lauragift21 I pushed again and removed the previous commit since it was not working properly. I removed the wrong diff in main.css file and fixed a bug in contact.md variable names.

**Comment by lauragift21** [comment]
Looks good to me too. :+1:

File: static-main/js/form.js [hunk 3]
```diff
@@ -13,13 +13,13 @@ if ($.validate) {
     validateOnEvent: true,
     onError: function($form) {
       $('form[id=' + $form.attr('id') +'] .generic-error').removeClass('d-none');
-  },
+    },
```
Comment (lauragift21) [inline static-main/js/form.js]: whitespace included here.

**Comment by mpourismaiel** [comment]
There's a debugger in the code. I don't have access to a laptop write now. I'll fix it in 3-4 hours.

File: static-main/js/form.js (hunk 3 above)
Comment (mpourismaiel) [reply static-main/js/form.js]: I don't understand.

Comment (mpourismaiel) [reply static-main/js/form.js]: No I added the whitespace myself. Previously the indent level did not match on this line, every other line started at the 3rd level and this line was not indented.
